vassal-doc/src/main/readme-referencemanual/ReferenceManual/GamePiece.adoc

```diff
@@ -108,9 +108,12 @@ Text Label _(Rotatable)_
 |Drawing Layers that will rotate should be *_above_* the Can Rotate and/or Can Pivot.
 
 |Can Rotate +
-Can Pivot
+Can Pivot +
+Mat Cargo
 |Can Rotate belongs below any drawing trait that it should rotate, but *_above_* any Can Pivot trait.
 
+Mat Cargo similarly belongs any drawing trait that it should cause to maintain relative facing.
+
```

Comment:
Missing the word "below" after "belongs".

Reply:
Thanks! That's an important word in the context, too!